integration-tests/src/test/java/tachyon/worker/BlockServiceHandlerIntegrationTest.java

```diff
@@ -275,7 +275,7 @@ public void requestSpaceTest() throws Exception {
     Exception exception = null;
     try {
       mWorkerServiceHandler.requestBlockLocation(SESSION_ID, blockId2, WORKER_CAPACITY_BYTES + 1);
-    } catch (OutOfSpaceException oose) {
+    } catch (WorkerOutOfSpaceException oose) {
```

Comment:
Declare the following public member for the test class:

```@Rule
  public final ExpectedException mException = ExpectedException.none();```

and then use ``mException.expect(WorkerOutOfSpaceException.class)`` here instead of the try / catch idiom.